package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@speakeasy-api/speakeasy-typescript-sdk",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
What's our stance on when this should get bumped? Do we bump with every change for the sdks? Should we be doing that for other things like embeds?

Reply:
So versions should be bumped with almost every change, I have been pushing for us to do this via https://www.conventionalcommits.org/en/v1.0.0/ which states things like bug fixes/refactors etc would bump the patch versions, features would bump minor version and breaking changes bump the major version

Reply:
And yes I believe we should version every external thing we release